### PR TITLE
Change API for TagsInput

### PR DESCRIPTION
**Pull request by AleshaOleg**

### CODE REVIEW

File: web/src/components/tags-input/index.tsx [hunk 1]
```diff
@@ -80,8 +70,8 @@ export const TagsInput: React.FC<TagsInputProps> = props => {
       value,
       onKeyDown: e => {
         onKeyDown?.(e)
-        if (e.key === 'Backspace' && !value.length && tags && tags.length) {
-          onRemove(tags[tags.length - 1].id)
+        if (e.key === 'Backspace' && !value.length && children?.length) {
+          onRemove(children[children.length - 1].props.id)
```
Comment (sk7) [inline web/src/components/tags-input/index.tsx:74]: onRemove for TagsInput is never set? I get `Uncaught TypeError: onRemove is not a function` when I press backspace

File: web/src/components/tags-input/index.tsx [hunk 2]
```diff
@@ -143,28 +133,9 @@ export const TagsInput: React.FC<TagsInputProps> = props => {
...
-                }}
-              >
-                {name}
-              </Tag>
-            )
-          })}
           {children &&
             children.map(child => (
-              <div key={genUid()} className={classes.tag}>
+              <div key={child.props.id} className={classes.tag}>
```
Comment (sk7) [inline web/src/components/tags-input/index.tsx:138]: why do we manually need to set an id on the `Tag`?

**Review by sk7 — CHANGES_REQUESTED** [review]

File: web/src/components/tags-input/index.tsx (hunk 2 above)
Comment (AleshaOleg) [reply web/src/components/tags-input/index.tsx:138]: Good point, thanks

File: web/src/components/tags-input/index.tsx (hunk 1 above)
Comment (AleshaOleg) [reply web/src/components/tags-input/index.tsx:74]: Yes, I added the `onRemove` function to the story. Thanks!

File: web/src/stories/tags-input/base.stories.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ export default {
 }
 
 export const Base: React.FC<TabsProps> = props => (
-  <TagsInput {...props}>
+  <TagsInput onRemove={() => alert('Remove')} {...props}>
```
Comment (sk7) [inline web/src/stories/tags-input/base.stories.tsx:12]: I think the TagsInout should actually provide this out of the box:
there should be no need to define `onRemove` here, but pressing backspace should automatically remove the last `Tag`

Comment (AleshaOleg) [reply web/src/stories/tags-input/base.stories.tsx:12]: Agree

**Comment by sk7** [comment]
in the story:
- pressing backspace to delete: deletes 👍 , shows alert message 👍 
- clicking "x" to delete a certain tag: does not delete ❌ , shows alert message 👍

**Pull request merged**